components/speedreader/speedreader_whitelist.h

```diff
@@ -48,6 +52,16 @@ class SpeedreaderWhitelist : public brave_component_updater::BraveComponent {
   base::WeakPtrFactory<SpeedreaderWhitelist> weak_factory_{this};
 };
 
+using OnComponentReadyCallback =
```

Comment:
what are these changes for?

Reply:
committed by accident, started to work on the other follow-up issues, will back this out